docs/DeveloperGuide.md

```diff
@@ -276,21 +276,21 @@ _{Explain here how the data archiving feature will be implemented}_
-| `* * *` | business owner            | choose which company and point-of-contact to delete | save only relevant companies and point-of-contact to remove clutter |
-| `* * *` | forgetful business owner  | see usage instructions                     | refer to instructions when I forget how to use the application             |
-| `* * *` | new business owner        | simply search for contacts                 | retrieve the only essential details i need                          |
-| `* * *` | new business owner        | view a company's details and point of contact | easily know who to contact for that company                        |
-| `* * *` | busy business owner       | quickly retrieve the data of the company   | save time without having to go through multiple layers             |
-| `* *`   | careless business owner   | edit the details of point-of-contact       | correct the mistakes that I did                                    |
-| `* *`   | efficient business owner  | sort the search result by price            | quickly know which company I have made the most transaction with    |
+| Priority | As a …​                  | I want to …​                                        | So that I can…​                                                     |
+|----------|--------------------------|-----------------------------------------------------|---------------------------------------------------------------------|
+| `* * *`  | business owner           | choose which company and point-of-contact to delete | save only relevant companies and point-of-contact to remove clutter |
+| `* * *`  | forgetful business owner | see usage instructions                              | refer to instructions when I forget how to use the application      |
+| `* * *`  | new business owner       | simply search for contacts                          | retrieve the only essential details i need                          |
+| `* * *`  | new business owner       | view a company's details and point-of-contact       | easily know who to contact for that company                         |
+| `* * *`  | busy business owner      | quickly retrieve the data of the company            | save time without having to go through multiple layers              |
+| `* *`    | careless business owner  | edit the details of point-of-contact                | correct the mistakes that I did                                     |
+| `* *`    | efficient business owner | sort the search result by price                     | quickly know which company I have made the most transaction with    |
 
 *{More to be added}*
 
 ### Use cases
 
-(For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
+(For all use cases below, the **System** is the `JeeqTracker` and the **Actor** is the `user`, unless specified otherwise)
 
 **Use case: Delete a person**
 
```

Comment:
Should this be Delete a company? Or are we not changing it yet? If we are, the lines below have to be changed as well.